### ISSUE TITLE
Uncaught TypeError: editor.getScopedSettingsDelegate is not a function

### ISSUE THREAD

**Issue by fkaleo** [nteract/hydrogen#1148]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.24.0-beta1 x64
**Electron**: 1.6.15
**OS**: Ubuntu 17.04
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.1.1

### Stack Trace

Uncaught TypeError: editor.getScopedSettingsDelegate is not a function

```
At /home/kaleo/.atom/packages/Hydrogen/lib/code-manager.js:127

TypeError: editor.getScopedSettingsDelegate is not a function
    at getRegexString (/packages/Hydrogen/lib/code-manager.js:127:14)
    at getCurrentCodeCell (/packages/Hydrogen/lib/code-manager.js:168:23)
    at Object.getCurrentCell (/packages/Hydrogen/lib/code-manager.js:236:10)
    at Object.runCell (/packages/Hydrogen/lib/main.js:504:40)
    at HTMLElement.hydrogenRunCellAndMoveDown (/packages/Hydrogen/lib/main.js:111:55)
    at CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app/src/command-registry.js:388:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom-beta/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom-beta/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom-beta/resources/app/src/window-event-handler.js:108:40)
```

### Commands

```
  2x -0:08.4.0 core:move-up (input.hidden-input)
     -0:08.0 hydrogen:run (input.hidden-input)
     -0:07.5.0 core:confirm (input.hidden-input)
  4x -0:06.8.0 core:move-down (input.hidden-input)
  2x -0:06.0 core:move-up (input.hidden-input)
     -0:05.3.0 hydrogen:run (input.hidden-input)
     -0:05.1.0 core:move-up (input.hidden-input)
     -0:04.8.0 hydrogen:run (input.hidden-input)
  2x -0:04.6.0 core:move-up (input.hidden-input)
     -0:03.9.0 editor:move-to-end-of-screen-line (input.hidden-input)
     -0:03.9.0 core:page-down (input.hidden-input)
     -0:03.3.0 core:page-up (input.hidden-input)
     -0:03.3.0 core:page-down (input.hidden-input)
  2x -0:03.0 core:page-up (input.hidden-input)
  2x -0:02.6.0 core:move-down (input.hidden-input)
     -0:02.1.0 hydrogen:run (input.hidden-input)
```

### Non-Core Packages

```
atom-ide-ui 0.7.0 
Hydrogen 2.1.1 
ide-python 0.3.0 
teletype 0.4.0 
```

**Comment by BenRussert** [comment]
Thanks for the report! This will be fixed on the next release, xref #1147

**Comment by lgeiger** [comment]
👍 We should do a release once #1144 and #1143 are merged

**Comment by lgeiger** [comment]
Fixed in 2.2.0
